blog/2022-12-14-dnsmasq-monitoring.md

```diff
@@ -0,0 +1,84 @@
+---
+slug: dnsmasq-monitoring
+title: How to monitor and troubleshoot Dnsmasq DNS Forwarder
+description: Monitoring and troubleshooting Dnsmasq DNS Forwarder.
+authors: satya
+tags: [how-to, monitor, troubleshoot, dnsmasq,  dns, domain-name-server, DNS-management
+ dhcp, network-management]
+keywords: [how-to, monitor, troubleshoot, dnsmasq,  dns, domain-name-server, DNS-management
+ dhcp, network-management]
+image: https://user-images.githubusercontent.com/96257330/207541929-5857a8fe-393e-492f-bc24-410ca76f0d9b.png
+
+---
+
+Find out how to effectively and easily monitor and troubleshoot Dnsmasq DNS Forwarder using Netdata
+
+<!--truncate-->
+## What is Dnsmasq?
+
+Dnsmasq is an open-source, lightweight, DNS caching and forwarding server. It is designed to provide DNS resolution for small and home networks. Dnsmasq provides local DNS caching, forwarding, and recursive lookups, as well as DHCP, TFTP, and other related services. It also has support for DNS and DHCPv6, as well as various other features such as DNS-over-TLS and IPv6 privacy extensions. Dnsmasq is a versatile and highly configurable tool that is simple to
+We will cover Dnsmasq_DHCP in a separate blog with more details.
```

Comment:
Should this really be a separate blog or just an addition here?

Reply:
It is a separate collector and a separate section in the TOC.. Although they are related.

Reply:
Just gonna add this here: Same as the other PR comment about saying "blog." Can we say "in a separate entry"?

Reply:
Fixed this.